Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,168 @@
+        // Get the new view controller using segue.destinationViewController.
+        // Pass the selected object to the new view controller.
+    }
+    */
+    
+    // MARK: - Table View Methods:
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int
+    {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int
+    {
+        return 8
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell
+    {
+        let cell = tableView.dequeueReusableCellWithIdentifier("emptyCell", forIndexPath: indexPath)
+
+        
+        if  indexPath.row == 0
+        {
+            let cell = tableView.dequeueReusableCellWithIdentifier("treeInfoCell", forIndexPath: indexPath)
+
+            cell.textLabel?.text = treeName
+            cell.imageView?.image = treeImage
+        }
+        if  indexPath.row == 1 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("locationCell", forIndexPath: indexPath)
+
+            cell.textLabel?.text = "Location:"
+        }
+        if  indexPath.row == 3 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("addressCell", forIndexPath: indexPath)
+
+            cell.textLabel?.text = address
+        }
+        if  indexPath.row == 4 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("cityStateZipCell", forIndexPath: indexPath)
+
+            cell.textLabel?.text = cityStateZip
+        }
+        if  indexPath.row == 5 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("contactCell", forIndexPath: indexPath)
+            
+            cell.textLabel?.text = "Contact:"
+        }
+        if  indexPath.row == 6 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("userNameCell", forIndexPath: indexPath)
+
+            
+            cell.textLabel?.text = name
+        }
+        if  indexPath.row == 7 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("phoneCell", forIndexPath: indexPath)
+
+            
+            cell.textLabel?.text = phone
+        }
+        if  indexPath.row == 8 {
+            
+            let cell = tableView.dequeueReusableCellWithIdentifier("emailCell", forIndexPath: indexPath)
+            
+            cell.textLabel?.text = email
+        }
+
+        
+        return cell
+    }
+    
+    // MARK: Action handlers:
+    
+    @IBAction func sendInformationTouchUpInside(sender: AnyObject)
+    {
+        let alertConfimationMessage = UIAlertController(title: "Confirmation", message: "Thank You! Your request has been sent", preferredStyle: .Alert)
+        let sentImage = UIImage(named: "sent")
+        let sentAction = UIAlertAction(title: "Ok", style: .Default, handler: nil)
+        sentAction.setValue(sentImage, forKey: "image")
```

Comment:
Don't force images into alerts as the look like the attached screenshot.
<img width="487" alt="screen shot 2016-10-20 at 9 08 53 am" src="https://cloud.githubusercontent.com/assets/1647012/19561031/043f115a-96a5-11e6-9cd2-9754d33c03bf.png">